js/main.js

```diff
@@ -1,2 +1,84 @@
-$(function () {
+$(() => {
+
+  // variables to keep track of the scores of
+  // the user and the computer
+  let userScore = 0
+  let botScore = 0
+
+  // defining other variables to
+  let message
+  const options = ["rock", "paper", "scissors"]
+
+  // listener for the click on one of the options
+  $("button").click(compare)
+
+  // compare user throw vs bot throw
+  function compare() {
+    const botThrow = randomizeBotOption()
+    // console.log(`the bot tossed ${botThrow}`)
+
+    // get userThrow
+    const userThrow = $(this).attr('id')
```

Comment:
Understanding "this" and implementing it the way you did is stepping into the world of advanced javascript; very impressive!